fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
It then comes down to how many developers would be attempting to us fsevents on a non-darwin system vs. how many existing dependencies would break on non-darwin systems because of the way they're parsing the exception.

I agree the message is entirely misleading in a vacuum, but in the context in which it's intended to be used, I would rather risk a developer being momentarily confused, which isn't necessary harmful in and of itself and easily mitigated with effective documentation and communication, to potentially breaking a great deal of things out there because of a change in behavior (in all likelihood, both are in low probability of occurring). 

What kind of issues would you rather be having to respond to? A couple of developers who attempted to use fsevents on win or linux, and were a little confused by the exception, vs. many that would be up in arms because their systems are no longer working?

If you have concurrence with this line of thinking, and can then step back to understand the ultimate purpose of this PR, can you offer any suggestions on a workable and acceptable solution, or provide any insight into exactly what conditions this repo requires, in order to accept a PR that serves the purpose of eliminating the constant install warning for fsevents from package managers, every time any dependency is added or removed on non-darwin systems? It would be greatly appreciated.